### PR TITLE
3.4.1 changelog entries / install version changes

### PR DESCRIPTION
**Pull request by dtrudg**
Add bug fix list to changelog

Update version number in install doc

### CODE REVIEW

**Review by WestleyK — COMMENTED** [review]
This looks good. Do you want to bump the go version from `1.12.6` to `1.12.9`? (in the INSTALL.md)

**Comment by dtrudg** [comment]
@WestleyK - that's a good point esp. as there were security fixes after `1.12.6`. Updated. Thanks.

**Review by WestleyK — APPROVED** [review]
Looks good to me :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,19 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
-# Changes Since v3.4.0
+# Changes Since v3.4.1
+
+# v3.4.1 - [2019.09.17]
+
```
Comment (jscook2345) [inline CHANGELOG.md:15]: I'd like to see links to PRs or Issues here, but I don't think it has to be done.

**Review by jscook2345 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dtrudg) [reply CHANGELOG.md:15]: I was thinking that too at first... as that was done in old 2.x changelogs I last was involved in.

However, in CHANGELOG.md we don't seem to do that now? I was following the precedent of the last point release... `3.1.1`, and there don't seem to be any links at all in the changelog these days

**Review by ikaneshiro — APPROVED** [review]

**Pull request merged**